kong/examples/kube/README.md

```diff
@@ -0,0 +1,82 @@
+---
+# Ingest metrics using the Kong integration
+
+The OTel Collector has a variety of [third party receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver) that provide integration with a wide variety of metric sources.
+
+Please note that not all metrics receivers available for the OpenTelemetry Collector have been tested by Lightstep Observability, and there may be bugs or unexpected issues in using these contributed receivers with Lightstep Observability metrics. File any issues with the appropriate OpenTelemetry community.
```

Comment:
we should replace Lightstep with "ServiceNow Cloud Observability" in all places since we are now ServiceNow Cloud Observability branding